dom-fetch-client/src/main/scala/org/http4s/dom/FetchReferrer.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.dom
+
+import org.http4s.Uri
+import org.http4s.util.Writer
+import org.http4s.util.Renderable
+
+sealed trait FetchReferrer extends Renderable
+object FetchReferrer {
+  final case object NoReferrer extends FetchReferrer {
+    override def render(writer: Writer): writer.type = writer << "no-referrer"
+  }
+  final case object Client extends FetchReferrer {
+    override def render(writer: Writer): writer.type = writer << "client"
```

Comment:
This be `about:client`.

In firefox, setting this to `client` causes the `Referer` to be `<BASE_URL>/client`, but `about:client` sets it to `<BASE_URL>/fetch` which I believe is the intended semantic.